hcn/hcnpolicy.go

```diff
@@ -157,20 +158,11 @@ type FiveTuple struct {
 	Priority        uint16 `json:",omitempty"`
 }
 
-// L4ProxyPolicySetting sets Layer-4 Proxy on an endpoint.
-type L4ProxyPolicySetting struct {
-	IP            string   `json:",omitempty"`
-	Port          string   `json:",omitempty"`
-	Protocol      uint32   `json:",omitempty"` // EX: TCP = 6, UDP = 17
-	ExceptionList []string `json:",omitempty"`
-	Destination   string   `json:","`
-	OutboundNat   bool     `json:",omitempty"`
-
-	// For the WFP proxy
-	FilterTuple   FiveTuple `json:",omitempty"`
-	ProxyType     ProxyType `json:",omitempty"`
-	UserSID       string    `json:",omitempty"`
-	CompartmentID uint32    `json:",omitempty"`
+// L4WfpProxyPolicySetting sets Layer-4 Proxy on an endpoint.
+type L4WfpProxyPolicySetting struct {
```

Comment:
Seems like a breaking change. All of these were exported previously why are they gone now?

Reply:
WHY: In an earlier design, we were trying to re-use a similar Policy object. But, there was very little overlap and it resulted in a cluttered mess. So I changed our design mid-way through Aurelien's work on an early POC.

RE BREAKING: This Proxy Policy ever only worked on certain _dev1 builds and is broken on anything recent. I have to fix up the SMI next.